pkg/dsync/drwmutex.go

```diff
@@ -237,8 +236,8 @@ func lock(ds *Dsync, locks *[]string, id, source string, isReadLock bool, lockNa
 		done := false
 		timeout := time.After(DRWMutexAcquireTimeout)
 
-		dquorum := int(len(restClnts)/2) + 1
-		dquorumReads := int(math.Ceil(float64(len(restClnts)) / 2.0))
+		dquorumReads := (len(restClnts) + 1) / 2
+		dquorum := dquorumReads + 1
```

Comment:
This is a bit pessimistic. For instance with 13 drives 7 drives is needed for read (>=50%) and 7 drives are needed for write (>50%).

```suggestion
		dquorum := (restClnts + 2) / 2
```

Reply:
Same reasoning we use to write atleast one extra parity, assumption is always hold +1 extra write lock to keep the symmetry. Otherwise it's hard to reason when even nodes occur we hold +1 but with odd node we relax. To keep the overall assumptions same.

Reply:
Yeah, let's keep the extra.